model/content_entity.go

```diff
@@ -32,9 +32,85 @@ func (w *ContentEntity) IsValid() bool {
 			return false
 		}
 	}
+
+	EnumValid := w.IsEnumValid()
+
+	if EnumValid == false {
+		return false
+	}
+	return true
+}
+
+//IsEnumValid this loops over the properties of an entity, if the enum is not nil, it will validate if the option the user set is valid
+//If nullable == true, this means "null" can be used as an option in the enumeration
```

Comment:
Also mention that the developer could pass in an empty string (it seems correct in the code so it's just to fix in the method comment)